packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md

```diff
@@ -0,0 +1,115 @@
+| `APP_ADMIN`                             | Application Administrator           | Apps                                  |
+| `GROUP_MEMBERSHIP_ADMIN`                | Group Membership Administrator      | [Groups](/docs/reference/api/groups/) |
+| `HELP_DESK_ADMIN`                       | Help Desk Administrator             | [Groups](/docs/reference/api/groups/) |
+| `MOBILE_ADMIN`                          | Mobile Administrator                |                                       |
+| `ORG_ADMIN`                             | Organizational Administrator        |                                       |
+| `READ_ONLY_ADMIN`                       | Read-Only Administrator             |                                       |
+| `REPORT_ADMIN`                          | Report Administrator                |                                       |
+| `SUPER_ADMIN`                           | Super Administrator                 |                                       |
+| `USER_ADMIN`                            | Group Administrator                 | [Groups](/docs/reference/api/groups/) |
+
+Standard role assignment is done in two steps:
+1. Assign a Role to a User or Group. At this point the admin has the supported privileges of the role over all resources across organization.
+2. (Optional) If the role supports targets, use one of the [target operations](/docs/reference/api/roles/#role-target-operations) to specify which specific resource the admin can manage.
+
+Note that the entities involved in standard role assignment are:
+* a Role; identified either by type or id returned from [listing API](/docs/reference/api/roles/#list-roles)
+* a Principal; either a Group or a User
+* (Optional) a resource; when using [target operations](/docs/reference/api/roles/#role-target-operations) this could be either an App or a Group
+
+## Custom Role Assignment
+Custom roles can be built by piecing [permissions](/docs/reference/api/roles/#permission-types) together. Once a custom role is built, you can use its `id` or `label` to assign to admins. The process is:
+1. Build a custom role
+2. Build a set of resources
+3. Bind the admin with the role from step 1 targeting the resource set from step 2
+
+An assignment of a Role to an admin is called a [binding](/docs/reference/api/roles/#binding-object). Conceptually, a Binding is identified by a unique id, and represents a single unique combination of Principal, Resource-Set, and Custom Role. A given Resource-Set can have multiple bindings, allowing for different combinations of principals and roles as necessary to grant permission to the encompassing resource.
+
+Therefore, when dealing with custom roles, three entities are always in play:
+* a Role; identified by its `label` or `id`
+* a Principal; either a Group or a User - known as a Member of the binding
+* a Resource-Set; identified by its `id`
```

Comment:
Sentence case for each bullet item.
Instead of semi-colon, use a colon in each bullet item and then initial caps after the colon.
Lowercase "role", "principal" and "resource", since it's lowercase in most places. If they should be uppercase, please make consistent.

* A role: Identified by its...
* A principal: Either a...
* A resource-set: Identified by...